include/zephyr/bluetooth/audio/csis.h

```diff
@@ -329,6 +329,9 @@ struct bt_csis_client_cb {
 	/* Device specific callbacks */
 	bt_csis_client_discover_cb             discover;
 	bt_csis_client_ordered_access_cb_t     ordered_access;
+
+	/** Internally used field for list handling */
+	sys_snode_t node;
```

Comment:
If the field is used internally only, then I'd suggest to prefix it with `__` so that `__node`

Reply:
In Zephyr we have primarily using a single underscore, so this will become `_node`.

Reply:
Double underscore is used as well, see Network Buffer API. Anyway, single or double - should be clear that it's for the internal use only.